x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_chart.tsx

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  AnnotationDomainTypes,
+  Axis,
+  AxisConfig,
+  Chart,
+  LineAnnotation,
+  LineAnnotationDatum,
+  LineSeries,
+  PartialTheme,
+  Position,
+  RecursivePartial,
+  ScaleType,
+  Settings,
+} from '@elastic/charts';
+import { EuiIcon } from '@elastic/eui';
+
+import React, { useCallback, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { DecisionPathPlotData } from './use_classification_path_data';
+
+const baselineStyle = {
+  line: {
+    strokeWidth: 1,
+    stroke: 'gray',
+    opacity: 1,
+  },
+  details: {
+    fontSize: 12,
+    fontFamily: 'Arial',
+    fontStyle: 'bold',
+    fill: 'gray',
+    padding: 0,
+  },
+};
+
+const axes: RecursivePartial<AxisConfig> = {
+  tickLabelStyle: {
+    fontSize: 12,
+  },
+};
```

Comment:
```suggestion
import euiVars from '@elastic/eui/dist/eui_theme_light.json';
const { euiColorFullShade, euiColorMediumShade } = euiVars;
const axisColor = euiColorMediumShade;

const baselineStyle: LineAnnotationStyle = {
  line: {
    strokeWidth: 1,
    stroke: euiColorFullShade,
    opacity: 0.75,
  },
  details: {
    fontFamily: 'Arial',
    fontSize: 10,
    fontStyle: 'bold',
    fill: euiColorMediumShade,
    padding: 0,
  },
};

const axes: RecursivePartial<AxisConfig> = {
  axisLineStyle: {
    stroke: axisColor,
  },
  tickLabelStyle: {
    fontSize: 10,
    fill: axisColor,
  },
  tickLineStyle: {
    stroke: axisColor,
  },
  gridLineStyle: {
    horizontal: {
      dash: [1, 2],
    },
    vertical: {
      strokeWidth: 0,
    },
  },
};
```

A suggestion with some style tweaks and using colors supplied by EUI:

<img src="https://user-images.githubusercontent.com/230104/91152948-e1396b00-e6bf-11ea-9aa5-2216b6c9f1c2.png" width="50%" />

Reply:
Updated here https://github.com/elastic/kibana/pull/73561/commits/9aa5d818f811def50962940b91d320a574af245a